src/watchmaker/workers/yum.py

```diff
@@ -25,15 +25,15 @@ class Yum(WorkerBase, LinuxPlatformManager):
 
     """
 
-    SUPPORTED_DISTS = ('amazon', 'centos', 'red hat')
+    SUPPORTED_DISTS = ('amazon', 'centos', 'red hat', 'rhel')
 
     # Pattern used to match against the first line of /etc/system-release. A
     # match will contain two groups: the dist name (e.g. 'red hat' or 'amazon')
     # and the dist version (e.g. '6.8' or '2016.09').
     DIST_PATTERN = re.compile(
         r"^({0})"
         "(?:[^0-9]+)"
-        r"([\d]+[.][\d]+)"
+        r"([\d]|[\d]+[.][\d])"
```

Comment:
With the other changes, `DIST_PATTERN` is not used at all, so can be removed entirely.

Reply:
Was _thinking_ that, but "wasn't eating anything". 

Removed.